torch/utils/data/datapipes/iter/combining.py

```
@@ -132,6 +152,16 @@ def __init__(self, datapipe: IterDataPipe, num_instances: int, buffer_size: int
                 "please be aware of OOM at random places",
                 UserWarning
             )
+        if copy is None:
+            self.copy_fn = None
+        elif copy == "shallow":
+            self.copy_fn = copymodule.copy
+        elif copy == "deep":
+            self.copy_fn = copymodule.deepcopy
+        else:
+            self.copy_fn = None
+            warnings.warn(f"Unknown copy method `{copy}` reqested, will not perform copying.", UserWarning)
```

We should raise Error here to prevent unexpected behavior